.github/workflows/integration_tests.yml

```diff
@@ -32,12 +32,21 @@ jobs:
       matrix:
         # Batch test fixes to land later
         include:
-          - build_type: centaurGcpBatch
-            build_mysql: 5.7
-            friendly_name: Centaur GCP Batch with MySQL 5.7
+          - build_type: centaurPapiV2beta
+            build_mysql: 5.6
+            friendly_name: Centaur Papi V2 Beta with MySQL 5.6 (current Terra production)
           - build_type: centaurPapiV2beta
             build_mysql: 5.7
             friendly_name: Centaur Papi V2 Beta with MySQL 5.7
+          - build_type: centaurPapiV2beta
+            build_mysql: 8.0
+            friendly_name: Centaur Papi V2 Beta with MySQL 8.0 (required Terra production)
+          - build_type: centaurPapiV2beta
+            build_mysql: 8.4
+            friendly_name: Centaur Papi V2 Beta with MySQL 8.4 (aspirational Terra production)
+          - build_type: centaurGcpBatch
+            build_mysql: 5.7
+            friendly_name: Centaur GCP Batch with MySQL 5.7
```

Comment:
Should we also try 8.0 and 8.4 for Batch?

Reply:
Probably redundant but I guess it won't hurt. GHA minutes are free!